sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkOperations.java

```diff
@@ -116,6 +134,30 @@ public static <T, TContext> CosmosItemOperation getCreateItemOperation(
         );
     }
 
+    public static <T, TContext> CosmosItemOperation getCreateItemOperation(
```

Comment:
Same here - Why are we exposing a new public API? Is this only for testing or do we want to expose this to customers?

Reply:
removed it since realized it is not needed.